languages/ruby/spec/oso/polar/polar_spec.rb

```diff
@@ -216,7 +216,7 @@ def initialize(bar:, baz:)
       it 'handles no args' do
         stub_const('Foo', Class.new)
         subject.register_class(Foo, from_polar: -> { Foo.new })
-        id = subject.host.make_instance('Foo', initargs: {}, id: 1)
+        id = subject.host.make_instance('Foo', args: [], kwargs: {}, id: 1)
```

Comment:
Can we add two new test cases here, one that tests args w/o kwargs and one that tests both args and kwargs at the same time?

Reply:
Added here as well as to the python tests